scipy/special/_add_newdocs.py

```diff
@@ -6839,10 +6839,12 @@ def add_newdoc(name, doc):
     ncfdtridfd : Inverse of `ncfdtr` with respect to `dfd`.
     ncfdtridfn : Inverse of `ncfdtr` with respect to `dfn`.
     ncfdtrinc : Inverse of `ncfdtr` with respect to `nc`.
+    scipy.stats.ncf : Non-central F distribution.
 
     Notes
     -----
-    Wrapper for the CDFLIB [1]_ Fortran routine `cdffnc`.
+    This function calculates the CDF of the non-central f distribution using
+    the Boost Math C++ library [1]_.
 
     The cumulative distribution function is computed using Formula 26.6.20 of
```

Comment:
Strictly speaking, this probably isn't true, right?

<img width="1142" alt="image" src="https://github.com/user-attachments/assets/592133e7-c7d1-4759-9bf8-44be1aa59721">

This is fine fine as is, but I point it out because presumably Boost has a more efficient way of performing the computation for large values of the noncentrality parameter.

Reply:
I noticed this too. It wasn't even true for cdflib. It could be changed to "is defined" or "can be computed", but I'll leave it for now.